script/gather_stake_accounts.rb

```diff
@@ -24,6 +24,17 @@
 
   TotalRewardsUpdateService.new(p.payload[:network], p.payload[:stake_accounts_active]).call
 
+  if (StakeAccount.where(network: network).last&.epoch || 0) > \
```

Comment:
Also, what happens if last StakeAccount has not been updated - eg. because it is no longer active and no longer returned by Solana CLI? StakeAccount.last.epoch will then have an old epoch number, not current epoch. I have 15 records like this on my local machine (not the last one, but it could theoretically be the last one?). It will probably never happen in reality, but I think would be better to pull current epoch from EpochWallClock (EpochWallClock.by_network...first.epoch) instead of StakeAccount? And then compare it against histories